tests/tests/operation_tests.cpp

```diff
@@ -1297,20 +1340,36 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
             const auto &new_alice = db->get_account("alice");
             const auto &new_bob = db->get_account("bob");
 
-            BOOST_REQUIRE(new_alice.balance.amount.value == ASSET("5.000 GOLOS").amount.value);
-            BOOST_REQUIRE(new_bob.balance.amount.value == ASSET("5.000 GOLOS").amount.value);
+            BOOST_CHECK_EQUAL(new_alice.balance.amount.value,  ASSET("5.000 GOLOS").amount.value);
+            BOOST_CHECK_EQUAL(new_bob.balance.amount.value,  ASSET("5.000 GOLOS").amount.value);
+            validate_database();
+
+            BOOST_TEST_MESSAGE("--- Test invalid amount (less digits after dot)");
```

Comment:
"less digits" can mislead, actually here is created asset with symbol, unsupported by `transfer` operation (like `"2.00 GOLOS"` or `"1.0 ANY"`)